src/src/rx_main.cpp

```diff
@@ -1210,6 +1208,11 @@ void MspReceiveComplete()
 #endif
         break;
     case MSP_ELRS_MAVLINK_TLM: // 0xFD
+        if (config.GetSerialProtocol() != PROTOCOL_MAVLINK)
+        {
+            config.SetSerialProtocol(PROTOCOL_MAVLINK);
+            deferExecutionMicros(100, reconfigureSerial);
```

Comment:
I'll assume this will get fixed in the merge

Reply:
Yep